world_universities_and_domains.json

```diff
@@ -99351,18 +99351,6 @@
       ],
       "country": "Ukraine"
   },
-  {
-      "web_pages": [
-          "http://www.univer.kharkov.ua/"
-      ],
-      "name": "Kharkiv National University",
-      "alpha_two_code": "UA",
-      "state-province": null,
-      "domains": [
-          "univer.kharkov.ua"
-      ],
-      "country": "Ukraine"
-  },
```

Comment:
Removed the old record and added the old domain and website to the new record. The old website is redirecting to the new one.